unstructured/ingest/connector/astra.py

```diff
@@ -67,10 +69,13 @@ def astra_db_collection(self) -> "AstraDBCollection":
         if self._astra_db_collection is None:
             from astrapy.db import AstraDB
 
-            # Build the Astra DB object
+            # Build the Astra DB object.
+            # caller_name/version for AstraDB tracking
             self._astra_db = AstraDB(
                 api_endpoint=self.connector_config.access_config.api_endpoint,
                 token=self.connector_config.access_config.token,
+                caller_name=integration_name,
+                caller_version=integration_version,
```

Comment:
Are Astra changes relevant to our PR?

Reply:
If they got in unexpectedly, let's remove; if intentional, let's add explanations in the PR description and changelog